### DISCUSSION TITLE
nose si tengo permisos

### DISCUSSION THREAD

**Discussion by bautistajau10** [community/community#141013] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

quiero saber si tengo permisos para despues poner clonar un repositorio y ahi poder compilarlo pero nose y estoy trabado

**Comment by AESMatias** [comment]
Hola @bautistajau10, para clonar un repositorio, puedes ir a la URL, copiar el enlace de git del repositorio como se muestra en esta imagen: 

![image](https://github.com/user-attachments/assets/2c44cd22-cde3-4e38-9c72-ee1494f695f0)

Luego, en tu Terminal de comandos que varía según el sistema operativo que uses, escribes ```git clone <HTTPS_URL_COPIED>``` , pegando la URL que has copiado. Si un repositorio tiene visibilidad pública, puedes clonarlo, lo mismo no aplica a los repositorios privados o dentro de alguna organización, donde quizás necesites permisos adicionales (_**Puedes investigar sobre los permisos adicionales aquí: [Aquí](https://docs.github.com/es/issues/planning-and-tracking-with-projects/managing-your-project/managing-access-to-your-projects)**_. 

También, es importante declarar o revisar la licencia actual del repositorio, la que indica las "condiciones de uso y distribución", típicamente en un archivo llamado LICENSE o LICENSE.txt (_**Puedes investigar sobre las licencias aquí: [Aquí](https://docs.github.com/es/articles/licensing-a-repository)**_)

Para forkear un proyecto, el procedimiento es ir a la URL del repositorio en cuestión, y luego darle click en "fork", como se muestra en la siguiente imagen.

![image](https://github.com/user-attachments/assets/d1d66896-d893-4791-afe4-3f3e6eac4b0e)

Luego de forkear, puedes clonar ese repositorio forkeado mediante la Terminal de comandos, con el mismo procedimiento indicado al principio, pero ahora tendrás permisos de edición para realizar cambios sobre ese fork. Una vez estés satisfecho con los cambios, puedes hacer un `git push origin BRANCH` donde branch es la rama a la que te interesa pushear, normalmente llamada main o dev.

Espero haberte ayudado, cualquier cosa adicional me comentas, saludos!
